guide/daily-spending-wallet/sending.md

```diff
@@ -218,6 +218,10 @@ In certain situations, the Lightning wallet may not have enough channel liquidit
 
 This fee is dependent on how many other transactions are currently waiting to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
 
+Unfortunately, it is common for users to overpay on-chain fees by mistake. It is recommended that wallets should have mechanisms in place to avoid that this happens.
```

Comment:
```suggestion
Unfortunately, it is common for users to overpay on-chain fees by mistake. Wallets should have mechanisms in place to avoid this happening.
```